### ISSUE TITLE
Consider removing or replacing "crates" with "Dependi"

### ISSUE THREAD

**Issue by EthanPintoA** [swellaby/vscode-rust-pack#122]
## Extension Info

- **Extension Name**: Dependi
- **Link to Extension on Marketplace**: https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi

## Value

Dependi is the successor of [crates](https://marketplace.visualstudio.com/items?itemName=serayuzgur.crates) (now deprecated). According to crates' README, Dependi includes all of crates' features for "free always," offers additional features, and will continue to receive updates (unlike crates).

**Comment by Maykeye** [comment]
[How about no?](https://www.youtube.com/watch?v=lITBGjNEp08)

They have stupidly different licenses.  

> Dependi includes all of crates' features for "free always,"

Lies that have nothing to do with reality:

> 1. **Non-Commercial Use**: The Software may not be used for commercial purposes.
   Commercial purposes include, but are not limited to, selling, reselling, renting,
   leasing, or offering the Software as a service or part of a service for a fee.

If it was "always"  this wouldn't exist.  

Reddit thread about deprecation of crates includes several alternatives for "free always".

**Comment by EthanPintoA** [comment]
You’re right that Dependi is not open source, and that it uses a freemium/source-available model, which is quite different from crates’ MIT license. However, I don’t agree that the maintainer of crates is “lying” when claiming that Dependi includes all of crates' features for "free always." The non-commercial clause does not make the extension paid, nor does it prevent most developers from using it day-to-day.

Dependi’s license explicitly allows free use for personal, educational, and **internal business purposes**, which covers the majority of, if not all, individual developers and teams using the crates VS Code extension or this extension pack:

> Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to use the Software for personal, educational, or internal business purposes...

Based on the license text, the commercial restriction appears to be aimed at resale, redistribution, or offering Dependi as part of a paid service, rather than ordinary development use. You could argue the wording is ambiguous or that it’s not truly "open," but that’s different from saying it isn’t free to use.

If the maintainers decide not to include Dependi specifically because it is not open source, that’s completely reasonable. Alternatively, given that crates is deprecated and VS Code now warns users and recommends migrating to Dependi, it may still make sense to remove crates from the list regardless of whether Dependi is added.
